server.js

```diff
@@ -29,7 +30,7 @@ const JAVASCRIPT_PATH = staticify.getVersionedPath('/javascripts/application.js'
 // Define app views
 const APP_VIEWS = [
   path.join(__dirname, '/govuk_modules/govuk_template/views/layouts'),
-  path.join(__dirname, '/app/views')
+  path.join(__dirname, '/app/common/views')
```

Comment:
Could we just have this as `/app/` rather than keep adding more paths as we build it out?

Reply:
Thanks @jonheslop , I tried to use `/app` but it doesn't work:
I guess it doesn't work recursively if we define second path?

Reply:
Addressed in 8219eb2